src/SdssShape.cc

```diff
@@ -560,8 +574,9 @@ bool getAdaptiveMoments(ImageT const &mimage, double bkgd, double xcen, double y
      */
     if (shape->flags[SdssShapeAlgorithm::UNWEIGHTED.number]) {
         w11 = w22 = w12 = 0;
-        if (calcmom<false>(image, xcen, ycen, bbox, bkgd, interpflag, w11, w12, w22, &I0, &sum, &sumx, &sumy,
-                           &sumxx, &sumxy, &sumyy, NULL, negative) < 0 ||
+        double ignored;
```

Comment:
Curious why this is `ignored` in place of `sums4`.

Reply:
In the old code NULL was passed. One could rename it to sums4, but this is never used. 
The old code used pointers to pass optional parameters C-style.

Reply:
Hmm, may be still keep the variable name to `psums4` or `sums4` but just add a comment saying it is ignored? It'll prevent looking up the definition in the future to see what that NULL corresponds to.